src/runner.rs

```diff
@@ -249,19 +311,33 @@ impl TestConfig {
 
         let total_tests = outputs.len();
         let mut failing_tests = 0;
+        let mut updated_tests = 0;
         for result in &outputs {
-            if let Err(error) = result {
-                eprintln!("{}", error);
-                failing_tests += 1;
+            match result {
+                Ok(_) => {}
+                Err(error @ InnerTestError::TestUpdated { .. }) => {
+                    eprintln!("{}", error);
+                    updated_tests += 1;
+                }
+                Err(error) => {
+                    eprintln!("{}", error);
+                    failing_tests += 1;
+                }
             }
         }
 
         println!(
-            "ran {} {} tests with {} and {}\n",
+            "ran {} {} tests with {} and {}{}\n",
             total_tests,
             "golden".bright_yellow(),
             format!("{} passing", total_tests - failing_tests).green(),
             format!("{} failing", failing_tests).red(),
+            if updated_tests != 0 {
+                format!(" - {} updated", updated_tests)
```

Comment:
Instead of tacking on a `- N updated` at the end, we should update the entire prompt so that it reads more naturally. E.g. the case of 0 updated tests can remain `ran 5 golden tests with 3 passing and 2 failing`, and the N > 0 updated case can read `ran 5 golden tests with 3 passing, 0 failing, and 2 updated`.

I was thinking it may be impossible to have any test failures if we're automatically changing test failures to updated tests instead but it should still be possible if the test failed from an IO error or similar rather than the expected output differing.

Reply:
Another option to consider is now that we have the overwrite flag, we can add a message hinting users to use it if they have failing tests. E.g. something along the lines of
```rs
if failing_tests > 0 {
    println!("Looks like you have failing tests. Review the output of each and fix any unexpected differences. When finished, you can use the --overwrite flag to automatically write the new output to each test file");
}
```

Reply:
I love all of these ideas. Implemented. But I think we can do even better here. So I've taken some "artistic liberties" and only mentioned it if thing can actually be "fixed" with the magic flag. Gotta keep that noise down. :D